lstchain/image/tests/test_modifier.py

```diff
@@ -59,7 +59,7 @@ def test_calculate_noise_parameters(mc_gamma_testfile, observed_dl1_files):
         mc_gamma_testfile,
         observed_dl1_files["dl1_file1"]
     )
-    assert np.isclose(extra_noise_in_dim_pixels, 1.96, atol=0.01)
+    assert np.isclose(extra_noise_in_dim_pixels, 1.455, atol=0.01)
```

Comment:
Shouldn't we understand why the noise changed significantly?

Reply:
Yes, although as I commented in the commit, the value is probably not meaningful because of the poor stats...

Reply:
Ok, comparing charges it seems to be a missing pulse integration correction, should be easy to find,.

Reply:
Missing now? I.e. before it was made and now it is not?

Reply:
yes, but perhaps specific to that part of the code

Reply:
I think I know what it is (and probably is missing only for interleaved events)

Reply:
The reason is that the default for absolute_factor in DL1CameraCalibrationContainer has changed from 1 to None:

https://github.com/cta-observatory/ctapipe/blob/3f2178084855b3fc3d5cf68ad69665c1eebb4de2/ctapipe/containers.py#L462-L466

https://github.com/cta-observatory/ctapipe/blob/5a1340adaade6167ea9e9ea73ebd175614a6ebf9/src/ctapipe/containers.py#L590-L595

When it is None, neither the absolute factor nor the relative factor (where the correction for pulse integration set in the event source config is applied) are used